Add Agora Chat Release Notes (#2466)
AgoraIO/agora_doc_source: en-US/markdown/agora-chat/Understand/agora_chat_release_notes_web.md

This page provides release notes for the Agora Chat Web SDK.

## V1.0.7
#### New features

- Supports custom chat room attributes.

> - Adds the `onLog`  method to implement user log callbacks.

Reviewer: 这个 onLog 和下面的 getJoinedGroups 的新增参数也都没有看到对应的 guide 更新……

Author: onLog 不用再官网上体现。

getJoinedGroups 在 chat_group_web 文档中更新代码：

```javascript
 connection.getJoinedGroups({
                 pageNum: 1,
                 pageSize: 500,
                 needAffiliations: true,
                 needRole: true
  })
  ```

Reviewer: 原先guide中这个接口代码如下：
conn.getJoinedGroups().then(res => console.log(res))
麻烦确认下是不是直接替换成你提交的这个，因为有 conn. 到 connection 的变化有点奇怪。

> - Adds the `needAffiliations` and ` needRole ` parameters in the `getJoinedGroups` method to allow you to get the number of members in a group and your own group role. 

Reviewer: ```suggestion
- Adds the `needAffiliations` and `needRole` parameters in the `getJoinedGroups` method to allow you to get the number of members and your group role in the groups that you join. 
```

#### Improvements

- Split the SDK into several modules.
- Added in-line comments.
- Optimized the reconnection logic.

#### Bugs fixed

- An issue with compatibility with Internet Explorer. 

> - The issue of lacking the longest validity period of token.

Reviewer: ```suggestion
- An issue of lacking the longest validity period of token.
```

Author: 1
